docs/pages/tutorials/minimal/extend_schema.mdx

```diff
@@ -0,0 +1,341 @@
+However, this is part of manipulating the schema and therefore included in this tutorial.
+
+```solidity
+History.set(newValue, block.number, block.timestamp);
+```
+
+Set the value.
+All MUD tables have a `<table>.set` function with the parameters being the key schema fields in order and then the value schema fields in order.
+
+    </details>
+
+2. Run this command in `packages/contracts` to rebuild everything this package produces.
+
+   ```sh copy
+   pnpm build
+   ```
+
+## Update the user interface
+
+You can already run the application and see in the MUD Dev Tools that there is a `:History` table and it gets updates when you click **Increment**.
+Click the **Store data** tab and select the table **:History**.
+
+However, you can also add the history to the user interface.
+The directions here apply to the `vanilla` user interface, if you use anything else you'll need to modify them as appropriate.
+
+1. Install an additional package.
+   Run this command in `packages/client`.
+
+   ```sh copy
+   pnpm install @latticexyz/store-sync@main
+   ```
+
+1. Edit `packages/clients/index.html`.
+
+   <CollapseCode>
+
+   ```html filename="index.html" copy showLineNumbers {12-26}
+   <!DOCTYPE html>
+   <html lang="en">
+     <head>
+       <meta charset="UTF-8" />
+       <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+       <title>a minimal MUD client</title>
+     </head>
+     <body>
+       <script type="module" src="/src/index.ts"></script>
+       <div>Counter: <span id="counter">0</span></div>
+       <button onclick="window.increment()">Increment</button>
+       <hr />
+       <h2>
+         History for value:
+         <select id="historyValue" onInput="window.readHistory(value)"></select>
+       </h2>
+       <table border>
+         <tr>
+           <th>Block number</th>
+           <th>Time</th>
+         </tr>
+         <tr>
+           <td id="blockNumber"></td>
+           <td id="timeStamp"></td>
+         </tr>
+       </table>
+     </body>
+   </html>
+   ```
+
+   </CollapseCode>
+
+<details>
+<summary>Explanation</summary>
+
+```html
+<hr />
+<h2>
+  History for value:
+  <select id="historyValue" onInput="window.readHistory(value)"></select>
+</h2>
+```
+
+This is the [input field](https://www.w3schools.com/tags/tag_select.asp) that lets the user select which counter value they'd like to get information about.
+The `id` attribute will be used by `packages/client/src/index.ts` to set the options. The `onInput` attribute is the JavaScript code to execute when the value changes.
+
+```html
+<table border></table>
+```
+
+A standard [HTML table](https://www.w3schools.com/html/html_tables.asp).
+
+```html
+<tr>
+  <th>Block number</th>
+  <th>Time</th>
+</tr>
+<tr>
+  <td id="blockNumber"></td>
+  <td id="timeStamp"></td>
+</tr>
+```
+
+A location for the values, which will be set by `window.readHistory` in `index.ts`.
+
+```html
+   </table>
+```
+
+Terminate the table in general.
```

Comment:
```suggestion
HTML closing tag for the table.
```
I'm not sure if we even need this line of explanation tbh, feel like we should expect basic html knowledge

Reply:
You're right. I tend to try and explain everything, but this is overkill.